rcl/include/rcl/security_directory.h

```diff
@@ -53,7 +53,7 @@ extern "C"
  * \returns machine specific (absolute) node secure root path or NULL on failure
```

Comment:
Mind adding a note here saying the returned pointer must be deallocated by the user?

Reply:
done.